File: 프로그래머스/Level2/JadenCase 문자열 만들기.py
# https://programmers.co.kr/learn/courses/30/lessons/12951

def solution(s):
    words = s.split(' ')
    answer = ''
    for word in words:
        for i in range(len(word)):
            if word[i].isalpha() == False:
                answer += word[i]
                continue
            if i == 0:
                answer += word[i].upper()
            else:
                answer += word[i].lower()
        answer += ' '
    return answer[:-1]

File: 프로그래머스/Level2/JadenCase 문자열 만들기.js
// https://programmers.co.kr/learn/courses/30/lessons/12951

const solution = (s) =>
	s
		.split(' ')
		.map((word) => word.toLowerCase())
		.map((word) => word.split(''))
		.map((word) => {
			if (word[0]) word[0] = word[0].toUpperCase();
			return word.join('');
		})
		.join(' ');
